src/cli-flags-helper.js

```diff
@@ -0,0 +1,38 @@
+const targetList = new Set([
+  "acorn",
+  "babel",
+  "babylon",
+  "buble",
+  "chai",
+  "coffeescript",
+  "espree",
+  "esprima",
+  "jshint",
+  "lebab",
+  "prepack",
+  "prettier",
+  "source-map",
+  "typescript",
+  "uglify-es",
+  "uglify-js"
+]);
+
+function getOnlyFlag(argv) {
+  if (process.argv.indexOf("--only") != -1) {
```

Comment:
Tip: `includes` instead of `indexOf`

Reply:
…unless you prefer to cache the result here and then reuse it for the `return` statement